### PR TITLE
Add water observed attributes for vigithau

### PR DESCRIPTION
**Pull request by junjou**

### CODE REVIEW

File: jsonld-contexts/vigithau-attributes.jsonld [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (bobeal) [inline jsonld-contexts/vigithau-attributes.jsonld:1]: why not adding them in https://github.com/easy-global-market/smbt-data-models/blob/master/jsonld-contexts/water-observed-attributes.jsonld ?

Comment (junjou) [reply jsonld-contexts/vigithau-attributes.jsonld:1]: Yes Gael made me the same comment

Comment (gpoujol) [reply jsonld-contexts/vigithau-attributes.jsonld:1]: I agree for waterLevel and rainfall but tension also ?

File: jsonld-contexts/vigithau-attributes.jsonld [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    "@context": {
+        "waterLevel": "https://smartdatamodels.org/waterLevel",
+        "rainfall": "https://smartdatamodels.org/rainfall",
+        "voltage": "https://smartdatamodels.org/voltage",
```
Comment (AHABID) [inline jsonld-contexts/vigithau-attributes.jsonld:5]: an extra `,`

**Review by gpoujol — APPROVED** [review]

**Review by AHABID — APPROVED** [review]

**Pull request merged**